server/budgettracker.business/Validation.cs

```diff
@@ -23,7 +23,6 @@ public static bool IsAccountRegistrationRequestValid(UserRequestApiContract argu
 
         public static bool IsCreateBudgetRequestValid(CreateBudgetRequestContract arguments)
         {
-            Console.WriteLine(arguments.Name + " " + arguments.SetAmount + " " + arguments.Duration);
```

Comment:
lol :+1:

Reply:
Haha managed to split by in an earlier PR